Sources/SwiftDocC/Utility/FeatureFlags.swift

```diff
@@ -13,7 +13,10 @@ public struct FeatureFlags: Codable {
     /// The current feature flags that Swift-DocC uses to conditionally enable
     /// (usually experimental) behavior in Swift-DocC.
     public static var current = FeatureFlags()
-    
+
+    /// Whether or not experimental annotation of code blocks is enabled.
+    public var isExperimentalCodeBlockEnabled = false
```

Comment:
nit: As a public API I feel like this name is missing a word somewhere. Right now, this reads as "is code blocks enabled" which is misleading and isn't what the value represents. 

One way to clarify what this represents could be to add a word like "annotations" or "extensions" or similar.